lib/parse-message.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+const Client = require('./../model/client');
+const EE = require('events');
+const ee = new EE();
+const allUsers = [];
+//module for parsing client request
+ee.on('default', function(client, data) {
+  allUsers.forEach(client => {
+    client.socket.write(`${client.nickname}: ${data} is not a command`);
+  });
+});
+ee.on('error', (err) => {
+  throw err;
+});
+  // if a client sends a message that starts with `/nick some_name`
```

Comment:
Comments are great for your reference, but try to get in the habit of pushing up code that doesn't contain extraneous comments to get into best practices of deploying clean code.